src/components/Counter.js

```diff
@@ -27,29 +27,27 @@ const CounterArea1 = styled.div`
   margin-right: 5px;
 `;
 
-const CounterArea2 = styled.span`
-  color: ${props => (props.marwin ? "yellowgreen" : "red")};
-  font-size: 50px;
-  font-family: "Segment7Standard";
-  background-color: rgb(0, 0, 0);
-  width: fit-content;
-  padding: 6px;
-`;
-
 const CounterUnit = styled.div`
   color: white;
   margin-top: 8px;
 `;
 
 function Counter({ fontColor }) {
+  const [count, setCount] = useState(1020);
+
+  useInterval(() => {
+    if (count >= 1) {
+      setCount(count - 1);
+    }
+  }, 10);
+
   return (
     <CounterBoxWrapper>
       <CounterDescription>Nur noch:</CounterDescription>
       <CounterWrapper>
-        <CounterArea1 marwin={fontColor}>4</CounterArea1>
-        <CounterArea2 marwin={fontColor}>7</CounterArea2>
+        <CounterArea1 marwin={fontColor}>{count}</CounterArea1>
```

Comment:
that still is a strange name for a prop :) albeit being an excellent name of course